src/desktop/apps/article/components/App.js

```diff
@@ -4,69 +4,65 @@ import ReactDOM from 'react-dom'
 import { Article } from '@artsy/reaction-force/dist/Components/Publishing'
 import ArticleLayout from './layouts/Article'
 import { EditButton } from 'desktop/apps/article/components/EditButton'
+import { hot } from 'react-hot-loader'
```

Comment:
Why do we need to include `react-hot-loader` directly in our code now? 
Do we only need to wrap a top-level parent component or any component we want to enable hot reloading on?

Reply:
This is pretty annoying, I agree, but we were over-wrapping with the previous solution and it was incompatible in Babel 7. And yup, only top-level component; it cascades down the tree. I'm going to look into automating this somehow, but right now there aren't much options. 

(See https://github.com/gaearon/react-hot-loader/tree/next for more info)